theme/templates/pages/imprint.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+
+{% block title %}{% trans %}Imprint {% endtrans %}{% endblock %}
+
+{% block body %}
+
+<div id="imprint" class="docs-section container">
+  <h2><a class="headline-link" href="#imprint">{% trans %}Impressum / Imprint{% endtrans %}</a></h2>
```

Comment:
```suggestion
  <h2><a class="headline-link" href="#imprint">{% trans %}Imprint / Impressum{% endtrans %}</a></h2>
```
Same order of languages accross the page